File: backend/account/src/main/java/com/ssafy/account/api/request/transaction/RemittanceRequest.java
package com.ssafy.account.api.request.transaction;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class RemittanceRequest {
    private Long myAccountId; // 내 계좌의 PK
    private Long receiverAccountId; // 수령인 계좌의 PK
    private String password; // 내 계좌 비밀번호
    private Long remittanceAmount; // 송금액
}

File: frontend/Mobile/app/src/main/java/com/allforyou/app/RemittanceRequest.kt
package com.allforyou.app

data class RemittanceRequest(
    var myAccountId: Long,
    var receiverAccountId : Long,
    var password: String,
    var remittanceAmount : Long,
)
object RemittanceRequestManager {
    private var data: RemittanceRequest? = null
    var name: String = ""
        var rfid:String=""
    var linkedAccountNumber:String=""
    var accountNumber:String = ""
    var paymentId:Long=0L
    var linkedAccountId=0L
    var transferId:Long=0L

    fun initData(request: RemittanceRequest) {
        data = request
    }

    fun getInstance(): RemittanceRequest {
        if (data == null) {
            data = RemittanceRequest(0,0,"", 0)
        }
        return data!!
    }
}